src/mpid/common/shm/mpidu_shm_barrier.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "mpidu_shm_impl.h"
+#include "mpidu_shm_barrier.h"
```

Comment:
Why not move to `mpidu_init_shm.c` and change them to "static" functions?

Reply:
This was my original idea but then I opted for keeping these in separate files. I can add a commit to merge the two